libbpf-cargo/src/lib.rs

```diff
@@ -64,6 +64,8 @@
 )]
 #![deny(unsafe_op_in_unsafe_fn)]
 
+pub use build::CompilationOutput;
```

Comment:
This struct would make more sense in `lib.rs`, but the build process here compiles each of the modules twice - once in the context of `libbpf_cargo` (lib) and once in the context of `libbpf_cargo` (bin). This means the modules can't access structs declared in `lib.rs`, even though declaring them there would make the interface more explicit.

Reply:
@danielocfb ah you've fixed this, awesome! Do you prefer the `pub use` or declaring the struct in lib.rs?

Reply:
I think keeping it in `build.rs` is fine -- after all its pretty closely related to the "build" part.